test/support/acceptance.go

```diff
@@ -1,34 +1,125 @@
-			if strings.HasPrefix(v, "quay.io") ||
-				strings.HasPrefix(v, "registry.redhat.io") ||
-				strings.Contains(v, "@sha256:") {
-				images = append(images, v)
-			}
+func GetImageDefinitionsFromSnapshot(jsonData Snapshot) []string {
+	return []string{
+		jsonData.CertificateTransparencyGo.CertificateTransparencyGo,
+		jsonData.Cli.ClientServerCg,
+		jsonData.Cli.ClientServerRe,
+		jsonData.Cli.Cosign,
+		jsonData.Cli.Gitsign,
+		jsonData.FbcV413.FbcV413,
+		jsonData.FbcV414.FbcV414,
+		jsonData.FbcV415.FbcV415,
+		jsonData.Fulcio.FulcioServer,
+		jsonData.Operator.RhtasOperator,
+		jsonData.Operator.RhtasOperatorBundle,
+		jsonData.Rekor.BackfillRedis,
+		jsonData.Rekor.RekorCli,
+		jsonData.Rekor.RekorServer,
+		jsonData.RekorSearchUI.RekorSearchUI,
+		jsonData.Scaffold.Createctconfig,
+		jsonData.Scaffold.CtlogManagectroots,
+		jsonData.Scaffold.FulcioCreatecerts,
+		jsonData.Scaffold.TrillianCreatedb,
+		jsonData.Scaffold.TrillianCreatetree,
+		jsonData.Scaffold.TufServer,
+		jsonData.SegmentBackupJob.SegmentBackupJob,
+		jsonData.Trillian.Database,
+		jsonData.Trillian.Logserver,
+		jsonData.Trillian.Logsigner,
+		jsonData.Trillian.Redis,
+	}
+}
+
+func GetCorrespondingSnapshotImage(operatorImageKey string, snapshotJsonData Snapshot) string {
+	switch operatorImageKey {
+	case "backfill-redis-image":
+		return snapshotJsonData.Rekor.BackfillRedis
+	case "client-server-cg-image":
+		return snapshotJsonData.Cli.ClientServerCg
+	case "client-server-re-image":
+		return snapshotJsonData.Cli.ClientServerRe
+	case "ctlog-image":
+		return snapshotJsonData.CertificateTransparencyGo.CertificateTransparencyGo
+	case "fulcio-server-image":
+		return snapshotJsonData.Fulcio.FulcioServer
+	case "rekor-redis-image":
+		return snapshotJsonData.Trillian.Redis
+	case "rekor-search-ui-image":
+		return snapshotJsonData.RekorSearchUI.RekorSearchUI
+	case "rekor-server-image":
+		return snapshotJsonData.Rekor.RekorServer
+	case "segment-backup-job-image":
+		return snapshotJsonData.SegmentBackupJob.SegmentBackupJob
+	case "trillian-db-image":
+		return snapshotJsonData.Trillian.Database
+	case "trillian-log-server-image":
+		return snapshotJsonData.Trillian.Logserver
+	case "trillian-log-signer-image":
+		return snapshotJsonData.Trillian.Logsigner
+	case "tuf-image":
+		return snapshotJsonData.Scaffold.TufServer
+	default:
+		return ""
+	}
+}
+
+func ParseOperatorImages(helpContent string) map[string]string {
+	re := regexp.MustCompile(`-(\S+image)\s+string[^"]+default "([^"]+)"`)
+	matches := re.FindAllStringSubmatch(helpContent, -1)
+
+	imageMap := make(map[string]string)
+	for _, match := range matches {
+		if len(match) > 2 {
+			key := match[1]
+			value := match[2]
+			imageMap[key] = value
 		}
 	}
-	return images
+	return imageMap
 }
 
-func HasValidSHA(imageDefinition string) bool {
-	const pattern = `[a-f0-9]{64}$`
-	re := regexp.MustCompile(pattern)
-	return re.MatchString(imageDefinition)
+func ValidateJson(filePath string) error {
```

Comment:
Not used

Reply:
It is used in 'json and yaml files are all valid' step